x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/curation/results/add_result_flyout.tsx

```diff
@@ -36,8 +37,10 @@ import { CurationLogic } from '../curation_logic';
 import { AddResultLogic, CurationResult } from './';
 
 export const AddResultFlyout: React.FC = () => {
-  const { searchQuery, searchResults, dataLoading } = useValues(AddResultLogic);
-  const { search, closeFlyout } = useActions(AddResultLogic);
+  const searchLogic = SearchLogic({ id: 'add-results-flyout' });
```

Comment:
To create `SearchLogic`, I basically factored it out of `AddResultsLogic`, and replaced the usage here of `AddResultLogic` with `SearchLogic`.